tools/identity-converter/Azure.Sdk.Tools.IdentityResolver.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
I think we should have the identity resolver and coverter in the same folder. If I'm understanding the difference, one is the library which we share and the other is a simple tool we wrap around the library.

Reply:
1. Both `notification-configuration` and `identity-converter` are using the common codes which includes the logic we read identity info from kusto database. 
2. If we put common codes in either folder, then the other has to reference common codes under another folder. 
3. As the tool developing, I believe we will have more and more shared codes which can pull in common. I think it is better to pull a common folder instead of reference from unrelated tool folder.

Reply:
I think it is OK to reference libraries across different tools directories, at least for now, it is no different then referencing it from a common directory. I look at this one very similar to the code-owners library and tool which live in the same tool directory and are referenced from other tools.